src/harness/testcases/WINNF_FT_S_SCS_testcase.py

```diff
@@ -64,3 +66,75 @@ def test_WINNF_FT_S_SCS_5(self):
     Checks that a CBSD registration with this configuration succeed.
     """
     self.doTestCipher('ECDHE-RSA-AES128-GCM-SHA256')
+
+  def generate_SCS_12_default_config(self, filename):
+    """Generates the WinnForum configuration for SCS.12"""
+    # Create the actual config for client cert/key path
+    
+    config = { 
+        'clientCert': self.getCertFilename("client_expired.cert"),
+        'clientKey':self.getCertFilename("client_expired.key")
+    }
+    writeConfig(filename, config)
+  
+  @configurable_testcase(generate_SCS_12_default_config)
+  def test_WINNF_FT_S_SCS_12(self,config_filename):
+    """Expired certificate presented during registration.
+
+    Checks that SAS UUT response with fatal alert message.
+    """
+    config = loadConfig(config_filename)
+    self.assertTlsHandshakeFailure(client_cert=config['client_cert_path'],
```

Comment:
The keys don't match those used in generate_SCS_12_default_config(). should be 'clientCert' and 'clientKey'

Reply:
Thanks   for catching this. The keys have been updated correctly.